src/strategies/base.ts

```diff
@@ -46,6 +46,7 @@ const DEFAULT_CHANGELOG_PATH = 'CHANGELOG.md';
 
 export interface BuildUpdatesOptions {
   changelogEntry: string;
+  commits: ConventionalCommit[];
```

Comment:
This may be a breaking interface change - strategies are a public class that can be extended. Make it optional?